test-integration/server/session/TransactionOLTPIT.java

```diff
@@ -335,6 +336,7 @@ public void whenShardingSuperNode_EnsureNewInstancesGoToNewShard() {
         assertThat(s3.links().collect(toSet()), containsInAnyOrder(s3_e1, s3_e2));
     }
 
+    @Ignore("flaky")
```

Comment:
why is sharding flaky?

Reply:
ask @lolski. I've already sent him my warmest regards :) and created an issue.